tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Quick question, should we upgrade this to `es6`?

Reply:
absolutely yes 👍